subprojects/gradle/instrumentation-test-impact-analysis/src/main/kotlin/com/avito/instrumentation/impact/InstrumentationTestImpactAnalysisPlugin.kt

```diff
@@ -39,8 +41,10 @@ class InstrumentationTestImpactAnalysisPlugin : Plugin<Project> {
                 group = TASK_GROUP
                 description = "Analyze androidTest bytecode to collect maps: [Screen:Test], [Screen:RootId]"
 
-                //todo we should also support flavors here
-                dependsOn("${project.path}:compile${it.testBuildType.capitalize()}AndroidTestKotlin")
+                val testBuildVariant = extension.testBuildVariant.convention(it.testBuildType)
+                    .map { "${project.path}:compile${it.capitalize(Locale.getDefault())}AndroidTestKotlin" }
```

Comment:
Should we distinguish build variants in `BytecodeResolver` and impact analysis logic ?
```
project.internalModule.getConfiguration(configurationType).let { configuration ->
            configuration.bytecodeSets() + configuration.dependencies.flatMap { it.fullBytecodeSets } <--
        }
```

Reply:
I check the ability to do that and the short answer we could but it creates a lot of work
So I propose to merge this version